webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -258,29 +243,30 @@ const WorkspaceDetails: React.FC = () => {
     tab && setTabKey(tab as WorkspaceDetailsTab);
   }, [workspaceId, navigate, tab]);
 
-  if (!workspace || Loadable.isLoading(loadableUsers)) {
+  if (loadableWorkspace === NotLoaded || Loadable.isLoading(loadableUsers)) {
```

Comment:
Why the inconsistency in how we're checking for NotLoaded?

Reply:
✔️ will change to `Loadable.isLoading`